swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md

```diff
@@ -0,0 +1,24 @@
+##How WES OpenApi 3.0 yaml was created from Swagger 2.0 yaml
+
+SwaggerHub was used to convert the WES 1.0.0 Swagger 2.0 yaml to OpenAPI 3.0 yaml
+
+####Steps to convert:
+1. Launch SwaggerHub https://app.swaggerhub.com/home
+2. Create an account and login to SwaggerHub
+3. Click on the 'plus' (**+**) icon on the left side of the page
+3. Select 'Import and Document API'
```

Comment:
Simpler to just put `1` all the way down

Reply:
fixed

Reply:
done